src/client/pages/Checkout/CheckoutDetails/QuoteDetails/QuoteDetails.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { colorsV3 } from '@hedviginsurance/brand'
+import styled from '@emotion/styled'
+import { getQuoteDetails } from 'utils/getQuoteDetails'
+import { useTextKeys } from 'utils/textKeys'
+import { OfferQuote } from '../../../OfferNew/types'
+import { SubSection } from '../components/SubSection'
+
+const { gray900, gray300, gray700 } = colorsV3
+
+const Row = styled.div`
+  font-size: 0.874rem;
```

Comment:
I think should be `0.875rem` :)
```suggestion
  font-size: 0.875rem;
```

Reply:
Thank you! My brain is really not functioning today 😵‍💫